Java/core/src/main/java/com/amazon/randomcutforest/imputation/ImputeVisitor.java

```diff
@@ -174,6 +182,13 @@ public void acceptLeaf(final INodeView leafNode, final int depthOfNode) {
             }
         }
 
+        if (centrality < 1.0) {
+            Random rng = new Random(randomSeed);
+            randomSeed = rng.nextLong();
+            rng.nextDouble(); // discard
```

Comment:
Why?

Reply:
I was attempting to decor relate a linear congruential pseudo random generator. But really this cannot be solved in this library. Removing.